csunplugged/templates/topics/programming-challenges-table.html

```diff
@@ -15,7 +15,9 @@
     {% for programming_challenge in programming_challenges %}
```

Comment:
(nowhere in particular)
The table stretches when the the details are expanded. When they're not expanded the content doesn't take up the full width of the container.

![plugging-it-in](https://user-images.githubusercontent.com/16066822/89594527-28dd8b80-d8a6-11ea-93a6-d76e7b01439e.gif)